ctapipe/reco/impact.py

```diff
@@ -204,8 +204,8 @@ def get_hillas_mean(self):
         tel_num = 0
 
         for hillas in self.hillas_parameters:
-            peak_x[tel_num] = hillas.x.to(u.rad).value  # Fill up array
-            peak_y[tel_num] = hillas.y.to(u.rad).value
+            peak_x[tel_num] = hillas.fov_lon.to(u.rad).value  # Fill up array
+            peak_y[tel_num] = hillas.fov_lat.to(u.rad).value
```

Comment:
Its not really x and y anymore, but I guess thats fine here

Reply:
It would be a small change to rename it but I'm really hesitant to do changes to the ImPACT code since #2052 is still open.

Reply:
Yeah, that is why I meant its fine.
Lets not touch IMPACT code more than we need to